src/cql-code-service.ts

```diff
@@ -1,7 +1,7 @@
 import { Code, ValueSet } from './datatypes/datatypes';
 import { TerminologyProvider, ValueSetDictionary, ValueSetObject } from './types';
 
-export class CodeService implements TerminologyProvider {
+class CodeService implements TerminologyProvider {
```

Comment:
In order to remain consistent with the rest of the code base, I think it's best to export the class declaration here (as it was before).